pybloqs/htmlconv/chrome_headless.py

```diff
@@ -0,0 +1,68 @@
+from io import open
```

Comment:
really excited to see this feature! I recall you speaking about it during the hackathon

Reply:
Yep, finally we'll get this merged!
Thanks again for all your contributions!!